checker/manual/external-tools.tex

```diff
@@ -302,17 +302,17 @@ \subsection{Explanation\label{ant-task-explanation}}
 \section{Maven\label{maven}}
```

Comment:
I found out that Maven seems to ignore annotation processors in some versions.... please make sure to use the latest version of the Maven compiler plugin, 3.3.